zephyr-apps/mppt/src/eFuse.c

```diff
@@ -0,0 +1,91 @@
+#include "eFuse.h"
+LOG_MODULE_REGISTER(mppt_eFuse, LOG_LEVEL_INF);
+
+//Defaults pin c14 to active at high voltage
+const struct gpio_dt_spec sig0 = GPIO_DT_SPEC_GET(DT_PATH(zephyr_user), signal_gpios);
+
+//Read E-Fuse pins
+int get_efuse_voltage(void)
```

Comment:
I would rename to `get_efuse_state` since that is representitive of what this function is actually doing. 

`gpio_pin_get_dt` returns the logic level: https://docs.zephyrproject.org/latest/hardware/peripherals/gpio.html#c.gpio_pin_get_dt